sdk/fixtures/src/main/java/com/klaviyo/fixtures/BaseTest.kt

```diff
@@ -58,14 +63,38 @@ abstract class BaseTest {
                 }
             }
         }
+
+        val htmlString = """
```

Comment:
Prefer to move this and its setup stuff to the one file that is using it. `BaseTest` is already on its way to getting a bit bloated, I prefer to only put properties and setup stuff in here if its used by multiple test files.